### ISSUE TITLE
cannot setup vhost to /public

### ISSUE THREAD

**Issue by redteaclub** [panique/mini2#44]
My webhosting company doesn't allow me to set vhost.

I just can upload the files to "public_html"

I try to setup the following .htaccess at public_html folder. 
It shows page not found. 

But I can access the page via http://www.abc.com/public/.

My web hosting server works fine for Mini1 for the following .htaccess
RewriteEngine on
RewriteRule ^(.*) public/$1 [L]

It does not work for Mini2

**Comment by panique** [comment]
hi, if you hoster doesnt allow vhost, then it's a very very bad or basic one :)
"public_html" looks like super-oldschool web hosting to me, this is not what you need here. Try to get a real server, it's super cheap and easy to setup these days.

**Comment by redteaclub** [comment]
Yes, I use this hosting over 10 years la. I am lazy to move all applications to Cloud platform. But thanks for your advice. I try to use mini3 :)
